examples/api_test/test_ssspapi.cu

```diff
@@ -0,0 +1,128 @@
+/*
+ * @brief Subgraph matching test for shared library advanced interface
+ * @file test_smapi.cu
+ */
+
+ #include <stdio.h>
+
+ #include <thrust/copy.h>
+ #include <thrust/host_vector.h>
+ #include <thrust/device_vector.h>
+ 
+ #include <gunrock/gunrock.h>
+ #include <gunrock/app/sssp/sssp_app.cu>
+ //#include <gunrock/app/test_base.cuh>
+ 
+ // __global__ 
+ // void kernel_print(unsigned long* list_subgraphs, unsigned long* size) {
+ //   int tid = threadIdx.x;
+ //   if(tid > size[0]) return;
+ //   printf("%lu\n", list_subgraphs[tid]);
+ // }
```

Comment:
Let's clean-up all the comments, remove any commented code that you don't need.

Reply:
I removed all deadspace/commented out code mentioned and ran the clang-format command as suggested, looks much cleaner now.